test/lib/mayaUsd/render/mayaToHydra/testMayaDisplayModes.py

```diff
@@ -0,0 +1,102 @@
+    _file = __file__
+
+    @property
+    def imageDiffFailThreshold(self):
+        # HYDRA-837 : Wireframes seem to have a slightly different color on macOS. We'll increase the thresholds
+        # for that platform specifically for now, so we can still catch issues on other platforms.
+        if platform.system() == "Darwin":
+            return 0.1
+        return 0.01
+    
+    @property
+    def imageDiffFailPercent(self):
+        # HYDRA-837 : Wireframes seem to have a slightly different color on macOS. We'll increase the thresholds
+        # for that platform specifically for now, so we can still catch issues on other platforms.
+        if platform.system() == "Darwin":
+            return 4
+        return 0.2
+
+    def switchDisplayModes(self):
+        panel = mayaUtils.activeModelPanel()
+        
+        cmds.modelEditor(panel, edit=True, wireframeOnShaded=False)
+
+        #Use Default Material
+        cmds.modelEditor(panel, edit=True, useDefaultMaterial=True)
+        cmds.refresh()
+        self.assertSnapshotClose("defaultMaterial" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+
+        #WireframeOnShaded mode
+        cmds.modelEditor(panel, edit=True, wireframeOnShaded=True)
+        cmds.refresh()
+        self.assertSnapshotClose("wireframeOnShaded" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+        cmds.modelEditor(panel, edit=True, wireframeOnShaded=False)
+
+        #Shadows mode
+        cmds.modelEditor(panel, edit=True, shadows=True)
+        cmds.refresh()
+        self.assertSnapshotClose("shadows" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+        cmds.modelEditor(panel, edit=True, shadows=False)
+
+        #All Lights mode
+        cmds.modelEditor(panel, edit=True, displayLights="all")
+        cmds.refresh()
+        self.assertSnapshotClose("allLights" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+
+        #xray mode
+        cmds.modelEditor(panel, edit=True, xray=True)
+        cmds.refresh()
+        self.assertSnapshotClose("xray" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+        cmds.modelEditor(panel, edit=True, xray=False)
+
+        #Display texures mode        
+        cmds.modelEditor(panel, edit=True, useDefaultMaterial=False)
+        cmds.modelEditor(panel, edit=True, displayTextures=True)
+        cmds.refresh()
+        self.assertSnapshotClose("displayTextures" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)     
+        cmds.modelEditor(panel, edit=True, displayTextures=False)
+
+        #Smoothshaded Material
+        cmds.modelEditor(panel, edit=True, displayAppearance="smoothShaded")
+        cmds.refresh()
+        self.assertSnapshotClose("smoothShaded" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+        
+
+    def test_MayaDisplayModes(self):
+        cmds.file(new=True, force=True)
```

Comment:
Just noticed this, but as an FYI, this is no longer necessary, as that command will now be run by default before each individual test (see [mtohUtils.py](https://github.com/Autodesk/maya-hydra/blob/dev/test/testUtils/mtohUtils.py#L61-L65)). Shouldn't cause issues though.

Reply:
noted. will incorporate it in my next PR.